scan_test.go

```diff
@@ -30,16 +30,20 @@ import (
 	"github.com/uber-go/dosa"
 )
 
-type AllTypes struct {
-	dosa.Entity `dosa:"primaryKey=BoolType"`
-	BoolType    bool
-	Int32Type   int32
-	Int64Type   int64
-	DoubleType  float64
-	StringType  string
-	BlobType    []byte
-	TimeType    time.Time
-	UUIDType    dosa.UUID
+type AllTypesScanTestEntity struct {
```

Comment:
Why was this renamed? Seems like having a duplicate name in a different package is a good test.